odl/tomo/analytic/filtered_back_projection.py

```diff
@@ -109,12 +285,22 @@ def fft_filter(x):
             scale = (ray_trafo.geometry.src_radius /
                      (ray_trafo.geometry.src_radius +
                       ray_trafo.geometry.det_radius))
+
+            if ray_trafo.geometry.pitch != 0:
+                # In helical geometry the whole volume is not in by each
+                # projection, ideally each point in the volume effects only by
+                # the projections in a half rotation.
```

Comment:
Check that sentence again. I think I understand what you mean, but there are several versions in one sentence :-)